src/http_server/api/v1/metrics.c

```diff
@@ -237,6 +318,11 @@ void cb_metrics_prometheus(mk_request_t *request, void *data)
                    PROMETHEUS_HEADER, sizeof(PROMETHEUS_HEADER) - 1);
     mk_http_send(request, sds, flb_sds_len(sds), NULL);
     mk_http_done(request);
+
```

Comment:
the following block must be always placed _before_ the call mk_http_done(), there are some cases where mk_http_done() can return right away (it's a coroutine and might yield)